actors/common/src/main/java/org/sunbird/learner/actors/bulkupload/BulkUploadBackGroundJobActor.java

```diff
@@ -972,6 +972,24 @@ private void processUserInfo(
     List<Map<String, Object>> failureUserReq = new ArrayList<>();
     List<Map<String, Object>> successUserReq = new ArrayList<>();
     Map<String, Object> userMap = null;
+    // To store hashTagId inside user-org table , first we need to get
```

Comment:
/* To store hashTagId inside user_org table, first we need to get hashTagId from provided organisation ID. Currently in bulk user upload, we are passing only one organisation, so we can get the details before for loop and reuse it. */